### PR TITLE
ViewF/ViewOptF/ViewListF rewrite + Tests + Update documentation.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,90 +1,111 @@
...
+[![Scala Steward badge](https://img.shields.io/badge/Scala_Steward-helping-blue.svg?style=flat&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAQCAMAAAARSr4IAAAAVFBMVEUAAACHjojlOy5NWlrKzcYRKjGFjIbp293YycuLa3pYY2LSqql4f3pCUFTgSjNodYRmcXUsPD/NTTbjRS+2jomhgnzNc223cGvZS0HaSD0XLjbaSjElhIr+AAAAAXRSTlMAQObYZgAAAHlJREFUCNdNyosOwyAIhWHAQS1Vt7a77/3fcxxdmv0xwmckutAR1nkm4ggbyEcg/wWmlGLDAA3oL50xi6fk5ffZ3E2E3QfZDCcCN2YtbEWZt+Drc6u6rlqv7Uk0LdKqqr5rk2UCRXOk0vmQKGfc94nOJyQjouF9H/wCc9gECEYfONoAAAAASUVORK5CYII=)](https://scala-steward.org) ![Build Status](https://github.com/rpiaggio/crystal/workflows/build/badge.svg) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.rpiaggio/crystal_2.13/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.rpiaggio/crystal_2.13)
 
-`import crystal._`
+`crystal` is a toolbelt to help build reactive UI apps in Scala by providing:
```
Comment (cquiroz) [inline README.md:5]: Nice, I like `toolbelt`

File: README.md [hunk 2]
```diff
@@ -1,90 +1,111 @@
...
+* `fold[B](fp: Long => B, fe: Throwable => B, fr: A => B): B`
+* `flatten[B]: Pot[B]` (if `A <: Pot[B]`)
+* `flatMap[B](f: A => Pot[B]): Pot[B]`
+* `toOption: Option[A]`
+* `toTryOption: Option[Try[A]]`
 
-### FixedLens
+The `crystal.data.implicits._` import will provide:
+* Instances for `cats` `MonadError`, `Traverse`, `Align` and `Eq` (as long as there's an `Eq[A]` in scope).
+* Convenience methods: `<Any>.ready`, `<Option[A]>.toPot`, `<Try[A]>.toPot` and `<Option[Try[A]]>.toPot`.
```
Comment (cquiroz) [inline README.md:40]: I wonder if `pure` would be a good alias for `ready` too

File: js/src/main/scala/crystal/data/react/implicits/package.scala [hunk 3]
```diff
@@ -51,4 +51,13 @@ package object implicits {
           }
       }
     )
+
+  implicit def viewReusability[F[_], A: Reusability]: Reusability[ViewF[F, A]] =
```
Comment (cquiroz) [inline js/src/main/scala/crystal/data/react/implicits/package.scala:55]: Thiss looks very useful

File: README.md (hunk 2 above)
Comment (rpiaggio) [reply README.md:40]: If I'm not mistaken, `pure` should work (provided by `cats`) if you have the implicit `Applicative[Pot]` in scope (`import crystal.data.implicits._`).

**Pull request merged**